### ISSUE TITLE
Make it possible to assign multiple codes to the same annotation

### ISSUE THREAD

**Issue by albertocottica** [edgeryders/annotator_store-gem#60]

**Comment by miahass** [comment]
In lieu of this, then fixing the difficulty of re-highlighting the same portion of text to add a new annotation is needed. I add the important because it's currently a frustration echoed by all of the ethnographers that creates frustration at high frequency every time they assign a code --- so I'm marking this as important because it's a high priority to do one or the other as soon as possible.

**Comment by tanius** [comment]
@miahass: If we implement this, one popup can create multiple annotations. The popup's annotation comment field would then have to be multiplied as well, providing one for each code selected in this popup. That would quickly overload the popup form (esp. combined with the ability to add translations on the fly, which may be added later, see the discussion in #137).

So what do we do with the annotation comment field then? I checked and it seems unused so far.

**Comment by miahass** [comment]
We can remove it! Erroneously thought that it would be a place to put code definitions while coding, but as you clarified a little while ago, it adds comments to the annotation not the code, so doesn’t get much use. Good hint :)

**Comment by damingo** [comment]
This is now implemented - but there is one limitation and one bug: 

Limitation: When an existing code is edited (click on the pencil in the popup) it is not possible to add more codes in the popup as this would create additional annotations. But we fixed the bug that the annotator button does not appear when the same section of a text is tagged multiple times. So it is now much easier to add additional codes to the same text section.

Bug: When multiple codes are added at once in the popover and one clicks "Save" and hovers with the mouse over the annotated text only the first code is visible in the popover. The other codes are only visible after a page reload.
